src/HexViewer.ts

```diff
@@ -256,17 +258,27 @@ export class HexViewer {
                 hexCell.cell.dataOffset = asciiCell.dataOffset = dataOffset;
 
                 var isSelected = this.selectionStart <= dataOffset && dataOffset <= this.selectionEnd;
-                $(hexCell.cell).toggleClass("selected", isSelected);
-                $(asciiCell).toggleClass("selected", isSelected);
+                var isCursor = dataOffset === this.selectionCursor;
+                var targets = [hexCell.cell, asciiCell];
+                $(targets).toggleClass("selected", isSelected);
+                $(targets).toggleClass("cursor", isCursor);
 
                 var skipInt = 0;
                 for (var level = 0; level < this.maxLevel; level++) {
                     var int = intervals[intIdx + level];
                     var intIn = int && int.start <= dataOffset && dataOffset <= int.end;
                     var intStart = intIn && int.start === dataOffset;
                     var intEnd = intIn && int.end === dataOffset;
-                    hexCell.levels[level].className = `l${this.maxLevel - 1 - level} ${((intBaseIdx + intIdx) % 2 === 0) ? "even" : "odd"}` +
-                        (intIn ? ` m${level}` : "") + (intStart ? " start" : "") + (intEnd ? " end" : "") + (isSelected ? " selected" : "");
+                    var classes = [
+                        `l${this.maxLevel - 1 - level}`,
+                        `${((intBaseIdx + intIdx) % 2 === 0) ? "even" : "odd"}`,
```

Comment:
String interpolation is unnecessary here.

```suggestion
                        (intBaseIdx + intIdx) % 2 === 0 ? "even" : "odd",
```